docs/book/src/topics/custom-images.md

```diff
@@ -6,13 +6,35 @@ This document will help you get a CAPZ Kubernetes cluster up and running with yo
 
 An *image* defines the operating system and Kubernetes components that will populate the disk of each node in your cluster.
 
-By default, images offered by "capi" in the [Azure Marketplace][azure-marketplace] are used. You can list these *reference images* with this command:
+By default, images offered by "capi" in the [Azure Marketplace][azure-marketplace] are used.
+
+You can list these *reference images* with this command:
 
 ```bash
 az vm image list --publisher cncf-upstream --offer capi --all -o table
 ```
 
-Note: These images are not updated for security fixes and it is recommended to always use the latest patch version for the Kubernetes version you wish to run. For production-like environments, and for more control over your nodes, it is highly recommended to build and use your own custom images.
+It is recommended to use the latest patch release of Kubernetes for a [supported minor release][supported-k8s].
+
+<aside class="note warning">
+
+<h1> Availability </h1>
+
+The Cluster API for Azure team publishes *reference images* for each Kubernetes release, for both Linux and Windows.
+
+Reference images for versions of Kubernetes that are no longer [supported by Cluster API][supported-capi] will be removed from the Azure Marketplace.
```

Comment:
I think in practice, we might want to keep more images around than this implies. @CecileRobertMichon also has a query that might be able to show us if a certain reference image has been used recently, as a further safety check before we actually delete them.

Reply:
Also I thought about justifying this statement with two reasons:

- Out-of-date versions of k8s often have unfixed CVEs or significant bugs, so making them always available is really a liability from our point of view
- It costs actual $$ to maintain this many images in storage

But I decided to omit the reasons, because I'm not sure it's necessary to justify it in the doc. But what do others think?

Reply:
I think we should stop providing images that we know are unpatched CVEs waiting to happen. We can warn people before we remove them, but we should remove them and stop providing them going forward, because it's a liability we do not need to carry. Cost isn't the main reason; "we aren't going to provide things people should not use" is a completely defensible reason.

The published statement can just be "we provide images of what k8s upstream is patching; here's a link to their release schedule, full stop".

Reply:
What if I modify it to say this:

> Reference images for versions of Kubernetes which have known security issues or which  are no longer supported by Cluster API will be removed from the Azure Marketplace.

(We do link to the k8s supported releases doc just above.)

Reply:
@bridgetkromhout I made the change above, let me know whatcha think.